### ISSUE TITLE
shutdowns giving "TypeError: Cannot redefine property: _token"

### ISSUE THREAD

**Issue by danclay** [danclay/eris-fleet#2]
When using shutdownCluster or totalShutdown an error occurs:
`TypeError: Cannot redefine property: _token`

This error is caused by a bug in eris. The bug is fixed in the [dev branch](https://github.com/abalabahaha/eris/tree/dev) of eris. Use `abalabahaha/eris#dev`. Once eris's dev branch is merged into the master branch eris-fleet's eris dependency will be updated.

**Comment by danclay** [comment]
Eris was updated to 0.13.2 in eris-fleet release 0.3.0
